locale/en/about/releases.md

```diff
@@ -18,3 +18,11 @@ schedule-footer: Dates are subject to change.
 ---
 
 # Releases
+
+<p>
```

Comment:
No need for the `<p>…</p>`, Handlebars adds paragraphs on its own. Otherwise LGTM.

Reply:
Thanks; just fixed that and force-pushed.